I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/prologue.rpy", line 41, in script call
    call pressButton
  File "game/prologue.rpy", line 68, in script call
    call meta_home
  File "game/Meta_Home.rpy", line 24, in script call
    call course_select
  File "game/Course_Select.rpy", line 7, in script
    jump course_end
ScriptError: could not find label 'course_end'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/prologue.rpy", line 41, in script call
    call pressButton
  File "game/prologue.rpy", line 68, in script call
    call meta_home
  File "game/Meta_Home.rpy", line 24, in script call
    call course_select
  File "game/Course_Select.rpy", line 7, in script
    jump course_end
  File "C:\Users\k1029\Desktop\Coding\renpy-7.4.11-sdk/renpy/ast.py", line 1757, in execute
    rv = renpy.game.script.lookup(target)
  File "C:\Users\k1029\Desktop\Coding\renpy-7.4.11-sdk/renpy/script.py", line 918, in lookup
    raise ScriptError("could not find label '%s'." % str(original))
ScriptError: could not find label 'course_end'.

Windows-10-10.0.19041
Ren'Py 7.4.11.2266
RenPy22 1.0
Mon Mar 14 17:04:42 2022
